openwrt-openwisp-monitoring/files/lib/openwisp/interfaces.lua

```diff
@@ -14,15 +14,23 @@ local ubus=ubus_lib.connect()
 if not ubus then
   error('Failed to connect to ubusd')
 end
+
 local interface_data=ubus:call('network.interface', 'dump', {})
 
 local interfaces={}
 
+local function popen(command)
+  local command_file = io.popen(command)
+  local output = command_file:read("*a")
+  command_file:close()
+  return output
+end
+
```

Comment:
duplicating the same function in each file is actually worse Kapil. If duplicating two lines of code is not good, how can duplicating 6 lines of code be better? Please rollback how it was before.